index.bs

```diff
@@ -11343,15 +11341,14 @@ in which case they are exposed on every object that [=implements=] the interface
             1.  Let |R| be <emu-val>null</emu-val>.
             1.  If |operation| is declared with a [{{Default}}] [=extended attribute=],
                 then:
-                1.  Set |R| be the result of performing the actions listed in
+                1.  Set |R| to the result of performing the actions listed in
                     |operation|'s [=corresponding default operation=] on |O|,
                     with |values| as the argument values.
             1.  Otherwise:
-                1.  Set |R| be the result of performing the actions listed in the
+                1.  Set |R| to the result of performing the actions listed in the
                     description of |operation|, on |O| if |O| is not <emu-val>null</emu-val>, with |values|
                     as the argument values.
-                1.  Return the result of [=converted to an ECMAScript value|converting=] |R| to
-                    an ECMAScript value of the type |op| is declared to return.
```

Comment:
I think this precision is sometimes useful and could lead to different results. At least long term I want to operate on a byte sequence and be able to return it, and let IDL handle the conversion to ByteString or Uint8Array as applicable.

Similar, if all you have is a list of things, you need the type to know what to turn that list into, no?

Reply:
But operations are not declared to return ECMAScript values, so the existing sentence is nonsensical.

Reply:
That is true, but the type is necessary somehow and this proposal removes it.

Reply:
How about something like this:

```diff
-             1.  Let |R| be <emu-val>null</emu-val>.
+             1.  Let |R| be an undefined value of the type |op| is declared to return.
```

Reply:
Maybe instead we should add an issue marker pointing to #674?